scala/pom.xml

```diff
@@ -79,33 +79,33 @@
       </dependency>
       <dependency>
           <groupId>org.apache.spark</groupId>
-          <artifactId>spark-core_2.10</artifactId>
+          <artifactId>spark-core_2.11</artifactId>
           <version>${spark.version}</version>
           <!--the following is placeholder for building uber package. Please keep as-is-->
 	      <!--<scope>provided</scope>-->
       </dependency>
       <dependency>
           <groupId>org.apache.spark</groupId>
-          <artifactId>spark-streaming_2.10</artifactId>
+          <artifactId>spark-streaming_2.11</artifactId>
           <version>${spark.version}</version>
           <!--the following is placeholder for building uber package. Please keep as-is-->
 	      <!--<scope>provided</scope>-->
       </dependency>
       <dependency>
           <groupId>org.apache.spark</groupId>
-          <artifactId>spark-streaming-kafka_2.10</artifactId>
+          <artifactId>spark-streaming-kafka-0-8_2.11</artifactId>
```

Comment:
Two different versions of this library exist to support two different versions of Kafka